a_star:
  start_time: 5
  obstacle_cost: 80